server/tskmgr/commander.go

```diff
@@ -0,0 +1,32 @@
+package tskmgr
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+)
+
+func Root(w http.ResponseWriter, r *http.Request) {
+	fmt.Printf("Got request on / endpoint")
```

Comment:
since we are building a server potentially hit by many different clients, we'd better use a log library to do the logging of what happens, logs record by default the timestamp which is crucial in debugging

```suggestion
	log.Printf("Got request on / endpoint\n")
```

Reply:
make sense. I have to see how logs are handled in GO. I'll probably ask you for some help

Reply:
It was easier than expected. I remember, in Python, my mind was blown